src/backend/vulkan/src/physical_device.rs

```diff
@@ -248,7 +248,8 @@ impl PhysicalDeviceFeatures {
             | Features::MUTABLE_COMPARISON_SAMPLER
             | Features::MUTABLE_UNNORMALIZED_SAMPLER
             | Features::TEXTURE_DESCRIPTOR_ARRAY
-            | Features::BUFFER_DESCRIPTOR_ARRAY;
+            | Features::BUFFER_DESCRIPTOR_ARRAY
+            | Features::EXTERNAL_MEMORY;
```

Comment:
Why are we exposing this unconditionally? It should be based off the presense of the extensions "DedicatedAllocation" and friends (everything that is required for external memory).

Reply:
Look to works correctly now. Please notify me if you still detect the problem and i will fix it as soon as possible.